File: 1968/1968.cs
public class Solution {
    public int ArrayWithElementsNotEqualToAve(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(ArrayWithElementsNotEqualToAve(root.left), ArrayWithElementsNotEqualToAve(root.right));
    }
}

File: 1968/1968.cpp
class Solution {
public:
    int arrayWithElementsNotEqualToAve(TreeNode* root) {
        if(!root) return 0;
        return 1+max(arrayWithElementsNotEqualToAve(root->left),arrayWithElementsNotEqualToAve(root->right));
    }
};